packages/calcite-components/src/components/split-button/split-button.scss

```diff
@@ -64,16 +64,19 @@
 :host([appearance="outline-fill"]) {
   &:host([kind="brand"]) {
     --calcite-split-button-divider: theme("colors.brand");
+    --calcite-split-button-background: transparent;
```

Comment:
Since the same value is applied regardless of `kind`, can you look at moving this declaration one level up?

```scss
:host([appearance="outline"]),
:host([appearance="outline-fill"]) {
  --calcite-split-button-background: transparent;

  &:host[kind="brand"]) {
    --calcite-split-button-divider: theme("colors.brand");
  }
  /* ... */
```

Also, this block overrides [these styles](https://github.com/Esri/calcite-design-system/pull/9298/files/b1f6fd6dd9ecf815c55fae7dc5d58c7dbbba1cbd..bbfc07ca4b2c06250166fa07ca18c2203b056409#diff-65b8f4931c47584fef5447263565a807eaca0e737c51acd4e7137a6b90079c6bR55-R61), so we can probably remove the overridden ones.

Reply:
Because of specificity, I can't factor out this line.

Reply:
Oh, I see. Disregard my comment then. Same with the disabled styling one below.

Sidebar: I think we could refactor this to simplify styles.